pandas/core/reshape.py

```diff
@@ -957,13 +957,15 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
         If `columns` is None then all the columns with
         `object` or `category` dtype will be converted.
     sparse : bool, default False
-        Whether the returned DataFrame should be sparse or not.
+        Whether the dummy columns should be sparse or not.  Returns
+        SparseDataFrame if `data` is a Series or if all columns are included.
```

Comment:
What about the case when some of the blocks are dense ?  e.g.
```
In [1]: import pandas as pd

In [2]: df = pd.DataFrame({'a':['A','B','C'],'b':['D','E','D']})

In [3]: pd.get_dummies(df, sparse=True, columns='b')
Out[3]: 
   a  b_D  b_E
0  A    1    0
1  B    0    1
2  C    1    0
```
Here `a` is still dense.  Is a df where some blocks are dense and some sparse more accurately called a `DataFrame` or a `SparseDataFrame`?